src/libraries/Common/src/System/Net/Http/aspnetcore/Http2/Hpack/HPackDecodingException.cs

```diff
@@ -21,6 +22,10 @@ public HPackDecodingException(string message, Exception innerException) : base(m
         {
         }
 
+#if NET8_0_OR_GREATER
```

Comment:
What's the ifdef needed for?

Reply:
I couldn't track it down either. As far as I could tell, each project that includes this file is only targeting `$(NetCoreAppCurrent)` TFMs.

1. [src\libraries\Common\tests\Common.Tests.csproj](https://github.com/dotnet/runtime/blob/main/src/libraries/Common/tests/Common.Tests.csproj#L5)
2. [src\libraries\System.Net.Http\src\System.Net.Http.csproj](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/System.Net.Http.csproj#L3)
3. [src\libraries\System.Net.Http\tests\UnitTests\System.Net.Http.Unit.Tests.csproj](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/tests/UnitTests/System.Net.Http.Unit.Tests.csproj#L6)

Reply:
@GrabYourPitchforks I tried removing this ifdef and the build still worked for me. I didn't check any of the other ifdefs though.

Reply:
@stephentoub @jeffhandley Good catch! It's probably left over from when I was trying to track down a compilation failure. Will remove.